x-pack/plugins/drilldowns/url_drilldown/public/lib/url_drilldown.tsx

```diff
@@ -90,21 +96,30 @@ export class UrlDrilldown implements Drilldown<Config, ActionContext, ActionFact
   }) => {
     // eslint-disable-next-line react-hooks/rules-of-hooks
     const variables = React.useMemo(() => this.getVariableList(context), [context]);
+
     return (
-      <UrlDrilldownCollectConfig
-        variables={variables}
-        config={config}
-        onConfig={onConfig}
-        syntaxHelpDocsLink={this.deps.getSyntaxHelpDocsLink()}
-        variablesHelpDocsLink={this.deps.getVariablesHelpDocsLink()}
-      />
+      <KibanaContextProvider
+        services={{
+          uiSettings: this.deps.uiSettings,
+        }}
+      >
+        <UrlDrilldownCollectConfig
+          variables={variables}
+          config={config}
+          onConfig={onConfig}
+          syntaxHelpDocsLink={this.deps.getSyntaxHelpDocsLink()}
+          variablesHelpDocsLink={this.deps.getVariablesHelpDocsLink()}
+        />
+      </KibanaContextProvider>
     );
   };
 
   public readonly CollectConfig = reactToUiComponent(this.ReactCollectConfig);
 
   public readonly createConfig = () => ({
-    url: { template: '' },
+    url: {
+      template: 'https://example.com/?{{event.key}}={{event.value}}',
```

Comment:
Thinking more on this, it might be wise to add in 'e.g. https://...' or 'Example: https://...' just to make it more clear that this is essentially placeholder.

Reply:
I've added "Example: "